### PR TITLE
Add modules for GraalVM 22.2.0

### PR DESCRIPTION
**Pull request by gastaldi**
/cc @cescoffier

### CODE REVIEW

**Review by zakkak — CHANGES_REQUESTED** [review]
Thanks for the update @gastaldi 

Note that you will also need to instruct github actions to build the new images by augmenting:
* https://github.com/quarkusio/quarkus-images/blob/main/.github/native-images.yaml
* https://github.com/quarkusio/quarkus-images/blob/main/.github/s2i-native-images.yaml
* https://github.com/quarkusio/quarkus-images/blob/main/.github/tooling-images.yaml

In the process please also remove older versions like 22.0.0

**Review by zakkak — APPROVED** [review]
LGTM. Thanks @gastaldi

**Comment by gastaldi** [comment]
Thank you @zakkak, now we wait for @cescoffier to review and merge this 😉

**Comment by galderz** [comment]
https://github.com/quarkusio/quarkus-images/pull/203 needs to be integrated before this PR can. Also, once integrated this PR might need to be reworked.

**Comment by galderz** [comment]
#203 has been integrated so the PR needs to be reworked.

**Comment by Karm** [comment]
Hello @gastaldi, could you add 21.3.3 and point 21.3 from 21.3.2 to 21.3.3 too, please?

**Comment by gsmet** [comment]
Let's try to be consistent with the work done here: https://github.com/quarkusio/quarkus-images/pull/202 .

**Comment by gastaldi** [comment]
> Hello @gastaldi, could you add 21.3.3 and point 21.3 from 21.3.2 to 21.3.3 too, please?

Hi @Karm! Sure, but I rather do that in a separate PR to avoid mixing things :)

File: modules/graalvm/22.2.0-java11/configure [hunk 1]
```diff
@@ -5,8 +5,7 @@ SOURCES_DIR=/tmp/artifacts
 
 ls -l ${SOURCES_DIR}
 tar xzf ${SOURCES_DIR}/${FILENAME} -C /opt
-ls -l /opt
-mv /opt/graalvm-ce-${GRAALVM_VERSION}* /opt/graalvm
+mv /opt/graalvm-ce-${GRAALVM_VERSION} /opt/graalvm
```
Comment (zakkak) [inline modules/graalvm/22.2.0-java11/configure:8]: @gastaldi if this didn't cause any issues I would prefer to keep it as it was.

Comment (gastaldi) [reply modules/graalvm/22.2.0-java11/configure]: Done

**Review by zakkak — APPROVED** [review]
LGTM. Thanks @gastaldi and sorry for the many iterations!

**Review by galderz — APPROVED** [review]

**Review by gsmet — APPROVED** [review]

**Pull request merged**